pkg/messaging/pubsub/server.go

```diff
@@ -61,3 +69,24 @@ func StartPubMessagingServer(ctx context.Context, nanomsgURL string, alerts <-ch
 		}
 	}
 }
+
+func replaceNodesWithAliases(ns *nodes.Storage, message string) (string, error) {
```

Comment:
The same issue as for `ReplaceNodesWithAliases`. And why this function is necessary if the same one already exists? It means that logic is separated to multiples places and that's not good.